From 3749600ebcafd13f25ed781e650bb3c838e5b7ad Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 11 Oct 2018 00:48:47 +0800
Subject: [PATCH 4809/5242] MLK-19870-1: ARM64: dts: imx8mm-evk: correct pmic
 name to bd71840

commit  af231e7d424c35cf5783b381f817d0f5b51b9ce2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Keep the same pmic driver name as the product name in board, although
bd71840 share the same driver as bd71837.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 8cc9240b..7974749 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -511,7 +511,7 @@
 
 	pmic: bd71837@4b {
 		reg = <0x4b>;
-		compatible = "rohm,bd71837";
+		compatible = "rohm,bd71840", "rohm,bd71837";
 		/* PMIC BD71837 PMIC_nINT GPIO1_IO3 */
 		pinctrl-0 = <&pinctrl_pmic>;
 		gpio_intr = <&gpio1 3 GPIO_ACTIVE_LOW>;
-- 
1.7.9.5

